Makefile

```diff
@@ -171,12 +171,13 @@ test-fullstack-unstable:
 .PHONY: test-with-database
 test-with-database:
 	test -d $(TEST_PG_PATH) && (pg_ctl -D $(TEST_PG_PATH) -s status >&/dev/null || pg_ctl -D $(TEST_PG_PATH) -s start) || ./t/test_postgresql $(TEST_PG_PATH)
-	PERL5OPT="$(PERL5OPT) -It/lib -I$(PWD)/t/lib -MOpenQA::Test::PatchDeparse" $(MAKE) test-unit-and-integration TEST_PG="DBI:Pg:dbname=openqa_test;host=$(TEST_PG_PATH)"
+	PERL5OPT="$(COVEROPT) $(PERL5OPT) -It/lib -I$(PWD)/t/lib -MOpenQA::Test::PatchDeparse" $(MAKE) test-unit-and-integration TEST_PG="DBI:Pg:dbname=openqa_test;host=$(TEST_PG_PATH)"
 	-[ $(KEEP_DB) = 1 ] || pg_ctl -D $(TEST_PG_PATH) stop
 
 .PHONY: test-unit-and-integration
 test-unit-and-integration:
 	export GLOBIGNORE="$(GLOBIGNORE)";\
+	export DEVEL_COVER_DB_FORMAT=JSON;\
```

Comment:
Doesn't that change the format for manually generated reports or does it only influence the intermediate format used by Devel::Cover itself?

Reply:
This is what's referred to [in Devel::Cover docs](https://metacpan.org/pod/Devel::Cover#Developer-variables) as `DB format`. Otherwise the format depends on the installed packages and is the reason #3751 was failing with weird JSON errors at one point.

Using it here also means you always get the same as what we use in CI, and you get an error if you don't have the dependencies needed to use that format.